TeeWhyKay/Chomp
# Remove/excessive emails

## Why

Friends complain too many emails.

## What

All emails removed except for results email where restaurant is decided.

## How to Test

Please help me go through the happy flow on your end before approving, thank you! Use a valid email so you can see if you are receiving other useless emails, and hopefully not.
You should only be receiving the restaurant results email.
